api/specs.proto

```diff
@@ -206,14 +206,14 @@ message EndpointSpec {
 		// network. The user may then provide an alternate
 		// option for ingress load balancing which is
 		// transparent to swarm.
-		DISABLED = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
+		LOCAL = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
```

Comment:
I am not sure what GLOBAL/LOCAL means here. DISABLED does not mean anything remotely related to LOCAL